examples/Four_Panel_Map.py

```diff
@@ -83,60 +82,51 @@ def plot_background(ax):
 
 ###########################################
 
-# Create the figure
-fig = plt.figure(figsize=(20, 15))
-add_metpy_logo(fig, 80, 100, size='large')
-gs = gridspec.GridSpec(5, 2, height_ratios=[1, .05, 1, .05, 0], bottom=.05, top=.95, wspace=.1)
+# Create the figure and plot background on different axes
+fig, axarr = plt.subplots(nrows=2, ncols=2, figsize=(20, 13), constrained_layout=True,
+                          subplot_kw=dict(projection=crs))
```

Comment:
CI is griping here about the `dict` call. What it really wants is to see `subplot_kw={'projection': crs}`

Reply:
I know that they are different ways to pass dictionaries in Python. This is the simplest form of the dictionary. Thanks for pointing this out. I will use this instead of the dict() format.

Reply:
The rationale behind it is that since they are equivalent, prefer `{}` since it is shorter and slightly faster. Not a big deal in this case, but best to be consistent.